airflow-core.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: airflow-core
+  version: 2.10.4
+  epoch: 0
+  description: Minimal Airflow core - Platform to programmatically author, schedule, and monitor workflows
+  dependencies:
+    runtime:
+      - bash
+      - busybox
+      - coreutils
+      - curl
+      - cyrus-sasl
+      - cyrus-sasl-dev
+      - findutils
+      - glibc-locale-en
+      - glibc-locales
+      - gnupg-utils
+      - gzip
+      - mariadb-dev
+      - mysql
+      - openssl
+      - postgresql-client
+      - postgresql-dev
+      - python-3.12
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - gcc
+      - glibc-dev
+      - mariadb-connector-c-dev
+      - mariadb-dev
+      - openssf-compiler-options
+      - pkgconf-dev
+      - postgresql-dev
+      - py3.12-build-bin
+      - py3.12-pip
+      - python-3.12
+      - python-3.12-dev
+      - wolfi-base
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/airflow
+      tag: ${{package.version}}
+      expected-commit: c083e456fa02c6cb32cdbe0c9ed3c3b2380beccd
+
+  - runs: |
+      # by default airflow celery provider is not built,but running the upstream helm chart requires it
```

Comment:
Theres more than just celery being inserted in patch.py. Can you share some more details about this requirement? Is there any upstream docs or Dockerfile we're referencing here? Also where should we check in future if there are issues maintaining this package and some of the settings in the python script may need to be changed?

Reply:
That's a good call, Mark, althought airflow-core will remain a minimal package so it might not even need this provider (that is something that we will know when we reach the stage of helm chart tests). My intention to have it here comes from the existing airflow package that reports this issue and verify if it works or not 
https://github.com/wolfi-dev/os/blob/66c102ad5a2b5defd16ce394e330b498efbaff4f/airflow.yaml#L50